vslib/src/sai_vs_interfacequery.cpp

```diff
@@ -921,3 +921,44 @@ sai_status_t sai_api_query(
             return SAI_STATUS_INVALID_PARAMETER;
     }
 }
+
+sai_status_t sai_query_attribute_enum_values_capability(
+        _In_ sai_object_id_t switch_id,
+        _In_ sai_object_type_t object_type,
+        _In_ sai_attr_id_t attr_id,
+        _Inout_ sai_s32_list_t *enum_values_capability)
+{
+    SWSS_LOG_ENTER();
+
+    if (object_type == SAI_OBJECT_TYPE_DEBUG_COUNTER && attr_id == SAI_DEBUG_COUNTER_ATTR_IN_DROP_REASON_LIST)
+    {
+        if (enum_values_capability->count < 3)
+        {
+            return SAI_STATUS_BUFFER_OVERFLOW;
+        }
+
+        enum_values_capability->count = 3;
```

Comment:
capabilities should come from saimetadata, since there is a way to add capability file when generating saimetadata from SAI/meta repo, but i think here we can skip it since it would be too hard for this pr to include that an refactor generating metadata independently for virtual switch

Reply:
I'll go ahead and make a note for this.